SU2_CFD/src/output/COutput.cpp

```diff
@@ -217,7 +217,7 @@ void COutput::SetHistory_Output(CGeometry *geometry,
 
   /*--- Output using only the master node ---*/
 
-  if (rank == MASTER_NODE && !noWriting) {
+  if (rank == MASTER_NODE && !noWriting && !config->GetMultizone_Problem()) {
```

Comment:
Won't this stop inner iterations from being written to history/screen?

Reply:
Yes, that was true. The first inner iteration for a new time iteration still updated the averaged value using the previous logic attempt if this was called. This is irrelevant now with the update to windowing directly, so I've reverted this.